elements/openlayers-core/ol-map.ts

```diff
@@ -19,6 +19,24 @@ import { forwardEvents } from './lib/events.js'
  * </ol-map>
  * ```
  *
+ * ### Setting the map container height
+ *
+ * It is expected that the map element will be wrapped in another element on your page.
+ * By default the map element has 100% height, meaning it will fill the parent element.
```

Comment:
That sentence is not telling the truth, which is actually the point. By default, the map will not have a height and OpenLayers actually sends warnings to the console informing about that.

One solution is to set an explicit height to the map itself. `flex` on the other hand is the cleanest way for the map to grow to fill its container, as shown below

Reply:
Thanks for clarifying 😄 
Do you think it's a bad idea to add `flex` by default to the component?
The allow the user to override if they need a fixed specific height

Reply:
Of course! That's actually what should be done. Sending a PR right away